src/transformState.js

```diff
@@ -5,7 +5,36 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const object of actions) {
+    const actionType = object.type;
```

Comment:
```suggestion
    const { type, extraData, keysToRemove } = object;
```
You can just get all the needed props using destructuring